components/HomeScreenComp/PopularCard/PopularCards.js

```diff
@@ -33,30 +34,24 @@ const PopularCards = () => {
           contentContainerStyle={{ marginTop: 15, gap: 16 }}
         >
           {/* Card code */}
+          {apiData.PopularWordCategory && apiData.PopularWordCategory[0] && (
           <PopularCard1
-            cardImage={require("../../../assets/popular_1.png")}
-            cardText={"Unstop's Creative Hackathon"}
+            cardImage={apiData.PopularWordCategory? apiData.PopularWordCategory[0].image : "https://d3nn873nee648n.cloudfront.net/900x600/20732/300-SM1072581.jpg"}
```

Comment:
Mridul , you have to loop around the card.  you should not duplicate the code for every card. use .map function or similar.